release-notes/topics/catalog-11-2022.adoc

```diff
@@ -0,0 +1,33 @@
+[[catalog-05-2022]]
+= Automation Services Catalog
+
+Automation Services Catalog provides an easy way to extend automation in your Ansible controller to a broader user base. It offers multi-level approval to ensure extending automation is done safely. And, Role Based Access allows you to restrict automation access to specified users. Automation Services Catalog is the shop window into the Ansible Automation Platform.
```

Comment:
Should 'Ansible Automation Platform' appear here without the article? Here's the naming conventions slide deck for reference: https://docs.google.com/presentation/d/1l3_R4ZZvBL4riI6d4tGZlAwob6gT7cxIHErl6RwuzLc/edit#slide=id.gca71490c38_1_754

Reply:
Hi Lizzi. Thanks for your comment. I intended to comment out this section from the master. adoc because there are no new updates for this section as far as I can see. It looks like I might have been missing a pair of //. So added it. We'll see if it works this time. Other than that, I was interested to see what you referred to here, but could not locate it in the document. Can you refer me to this item specifically (page #, etc.)?